packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -107,13 +154,27 @@ export class BlobManager {
             await new Promise<void>((res) => this.runtime.once("attached", res));
         }
 
+        if (this.runtime.connected === false &&
```

Comment:
> I think one of the intentions of this design should be - treat offline and really slow network uniformly

I agree it would be best to treat these scenarios the same. The design in this PR could be changed to treat offline and online the same, but this would mean adding to the redirect table for every blob upload, instead of only for blobs uploaded while detached or offline.

> as long as blob is pending upload, it should be blocking op processing.

Not sure what you mean by this, do you mean blocking op submission?